src/taxes/pages/TaxClassesPage/form.tsx

```diff
@@ -77,8 +77,13 @@ function useTaxClassesForm(
     name: data.name,
     [isNewTaxClass
       ? "createCountryRates"
-      : "updateCountryRates"]: formset.data.map(item => {
+      : "updateCountryRates"]: formset.data.flatMap(item => {
       const { id, value } = item;
+      // TODO: when backend allows removing rates by passing null,
+      // we can stop filtering out empty values
```

Comment:
Do we have issue for this todo in repo and can we reference it here?

Reply:
Actually it was just fixed so I'll remove this condition now